src/components/documents/DocumentHead.tsx

```diff
@@ -12,27 +12,30 @@ import { AlertCircleIcon } from "@components/svg/Icons";
 import { Alert } from "@components/Alert";
 import { ExternalLink } from "@components/ExternalLink";
 import { Heading } from "@components/typography/Heading";
+import { LinkWithQuery } from "@components/LinkWithQuery";
 
 import { getCountryName, getCountrySlug } from "@helpers/getCountryFields";
 import { truncateString } from "@utils/truncateString";
 
 import { MAX_FAMILY_SUMMARY_LENGTH_BRIEF } from "@constants/document";
 
-import { TDocumentPage, TFamilyPage } from "@types";
+import { TDocumentPage, TFamilyPage, TConcept, TSearchResponse } from "@types";
 import { DocumentMetaRenderer } from "./renderers/DocumentMetaRenderer";
+import { getFeatureFlags } from "@utils/featureFlags";
 
 type TProps = {
   document: TDocumentPage;
   family: TFamilyPage;
   handleViewOtherDocsClick: (e: React.FormEvent<HTMLButtonElement>) => void;
   handleViewSourceClick: (e: React.FormEvent<HTMLButtonElement>) => void;
+  concepts?: (TConcept & { count: number })[];
```

Comment:
suggestion: The type of this from pages/documents/[id] will always be an array. Using this should simplify the rest of the code a little.

```suggestion
  concepts: (TConcept & { count: number })[];
```